net.newpipe.NewPipe.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="vcs-browser">https://github.com/TeamNewPipe/NewPipe</url>
+  <url type="homepage">https://newpipe.net/</url>
+  <url type="bugtracker">https://gitlab.com/android_translation_layer/android_translation_layer/-/issues</url>
+  <url type="donation">https://newpipe.net/donate/</url>
+  <url type="translate">https://hosted.weblate.org/engage/newpipe/</url>
+  <description>
+    <p>NewPipe does not use any Google framework libraries, or the YouTube API. It only parses the website in order to gain the information it needs. Therefore this app can be used on devices without Google Services installed. Also, you don't need a YouTube account to use NewPipe, and it's FLOSS.</p>
+    <p>This is an unofficial Flatpak build based on Android Translation Layer (<a href="https://gitlab.com/android_translation_layer/android_translation_layer">https://gitlab.com/android_translation_layer/android_translation_layer</a>). Please report bugs to the ATL bug tracker instead of the NewPipe bug tracker.</p>
+    <p>ATL is a lightweight WINE like compatibility layer for running Android applications on GNU/Linux Freedesktop systems. With the help of ATL, NewPipe integrates nicely into the Linux Desktop environment. Including support for notification, media controls and automatic dark/light theme selection. The translation layer is already bundled into the Flatpak and and does not need to be installed separately.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/master/screenshots/newpipe/00.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/master/screenshots/newpipe/01.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/master/screenshots/newpipe/02.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/master/screenshots/newpipe/03.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/master/screenshots/newpipe/04.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/master/screenshots/newpipe/05.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/master/screenshots/newpipe/06.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/master/screenshots/newpipe/07.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>AudioVideo</category>
+  </categories>
+  <launchable type="desktop-id">net.newpipe.NewPipe.desktop</launchable>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
```

Comment:
release and content_rating tags are missing

Reply:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#release https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#oars-information

Reply:
done